lib/ui/recommendation-row.js

```diff
@@ -256,8 +256,10 @@ RecommendationRow.prototype = {
     const val = key.split('.').reduce(function(o, x) {
       return (typeof o == 'undefined' || o === null) ? o : o[x];
     }, this._data);
-    const isEmptyObject = (typeof val == 'object' && !Object.keys(val).length);
-    return (typeof val == 'undefined' || !val || isEmptyObject) ? null : val;
+    let isEmptyObject = val => {
+      return val && typeof val == 'object' && !Object.keys(val).length;
```

Comment:
Right. That covers the two substantive comments here, so I'm going to merge this. Followed up with https://github.com/mozilla/universal-search/issues/199, which we should do very shortly after launch.